Documentation/aws.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Running Rook in AWS
+weight: 25
+indent: true
+---
+
+# Running Rook in AWS
+
+Rook can functionally replace Elastic Block Storage (EBS), Elastic File System (EFS) and S3 with Rook ObjectStore. Potentially reducing storage cost, increasing performance and portability of your Kubernetes workloads.
+
+:information_source:  
+Please refer to [Creating Rook Clusters][3] on how to customize your cluster to use specific devices.
```

Comment:
This sentence seems out of place and doesnt go with the flow of the paragraph. Perhaps put it at the end?